### ISSUE TITLE
Installation throws warning about un-existent releases

### ISSUE THREAD

**Issue by Djiit** [WilixLead/iohook#43]
edit : see later discussions; this is related to non-existant arch in linux platform.
Can you point me toward a working windows version ?

I can't find any prebuilt for Win10 64bit (I assume it might be the most common use case) for ABI 59 (node 9.2.0+)

Maybe the README should tell the user that one can't build iohook using the npm release, you have to clone the repository. I managed to build iohook by first building libiohook (beware of crlf errors; use dos2unix (see https://github.com/kwhat/libuiohook/issues/24)).

edit: we can use this issue as a dedicated windows thread if it helps debugging, and then submit a PR with more details on how to use this on Windows.

**Comment by Djiit** [comment]
Maybe @matthewshirley can help here 🙏

**Comment by matthewshirley** [comment]
PR #42 was merged yesterday, and, a new release was created and published on npm: https://github.com/WilixLead/iohook/releases/tag/v0.1.13

I hope this has fixed most of the issues with Windows. Can you give it a go? 👍

**Comment by Djiit** [comment]
Hey, thanks for your answer. Reading the logs I figured out 2 things :
- it was linux 32b version missing (yep, at first I though it was my current OS missing)
- linux 32b builds are marked "x32", not "ia32". is it intentional ? It make the example unusable (it throws errors)
```
"iohook": {
  "targets": [
    "node-51",
    "electron-53"
  ],
  "platforms": [
    "win32",
    "darwin",
    "linux"
  ],
  "arches": [
    "x64",
    "ia32"
  ]
}
```

**Comment by Djiit** [comment]
PR https://github.com/WilixLead/iohook/pull/45 might help on this.

**Comment by Djiit** [comment]
Fixed in #45
